src/openforms/prefill/contrib/haalcentraal/plugin.py

```diff
@@ -67,6 +68,10 @@ def _get_values_for_bsn(cls, bsn: str, attributes: Iterable[str]) -> Dict[str, A
     def get_prefill_values(
         cls, submission: Submission, attributes: List[str]
     ) -> Dict[str, Any]:
+        # FIXME workaround because this method is a classmethod
+        config = GlobalConfiguration.get_solo()
+        if not config.plugin_enabled("haalcentraal"):
+            raise PluginNotEnabled()
```

Comment:
would also move this to the caller location :-) there's an `apply_prefill` somewhere in the codebase that calls this